BrainPortal/lib/cbrain_task_generators/templates/bourreau.rb.erb

```diff
@@ -438,6 +438,11 @@ class CbrainTask::<%= name %> < <%= (descriptor['custom'] || {})['cbrain:inherit
 %   end
       else # Check exit status value
         exit_status = status_file_content.to_i
+        # provide explanations for exit statuses (if any):
+        <% descriptor['error-codes']&.each do |err| %>
```

Comment:
Why not `descriptor.error_codes.each` ? and why the `&`

Reply:
I do not think `error_codes` method exists. Also, key does not necessary exists hence '&'. '&' is more concise than `descriptor['error-codes'] || [ ] ` hack

Reply:
I think error_codes and other helper method only available for new style boutiques